scripts/testnet/variables.sh

```diff
@@ -60,14 +60,14 @@ export GENESIS_STAKE_TYPE="direct" #'delegated' or 'direct' as in direct stake
 #if set to 1, each observer will turn off the antiflooding capability, allowing spam in our network
 export OBSERVERS_ANTIFLOOD_DISABLE=0
 
-# If set to true, this will deploy nodes in a sovereign shard.
+# If set to 1, this will deploy nodes in a sovereign shard.
 # All variables from metashard structure(validators, observers, consensus) should be set to zero and SHARDCOUNT to 1
 # For now, make sure that you checkout feat/sovereign branch from mx-chain-deploy repo when using these scripts
-export SOVEREIGN_DEPLOY=true
+export SOVEREIGN_DEPLOY=1
 
 # Shard structure
 export SHARDCOUNT=1
-export SHARD_VALIDATORCOUNT=2
+export SHARD_VALIDATORCOUNT=3
```

Comment:
We should not change this. the script to deploy sovereign full stack works with a min of 3 here.
Inside `scripts/testnet/deployFullStackSovereignChainWithScreen.sh`, when updating variables we need to check if `SHARD_VALIDATORCOUNT` is less than 3, and if so, set to it to 3, otherwise leave it unchanged